Cargo.toml

```diff
@@ -36,6 +36,7 @@ thiserror = "1"
 unicode-width = "0.2"
 os_pipe = "1.1.5"
 libc = "0.2.155"
+iceoryx2 = "0.4.1"
```

Comment:
This requires rust 1.75 or newer, but CI runs rust 1.74.
A rust-toolchain file and Cargo.toml entry would help make this more visible.